addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -399,6 +410,9 @@ const Wysiwyg = Widget.extend({
     redo: function () {
         this.odooEditor.historyRedo();
     },
+    focus() {
```

Comment:
@Goaman t'a dupliqué la fonction focus(). Probablement dans une résolution de conflit ou quoi. 

Faudra corriger ça.

Reply:
@sebgeelen let's try to stick to english as the official discussion language please :)

Reply:
#71319